src/main/java/com/example/schedule/demo/service/ScheduleImpl.java

```diff
@@ -20,5 +22,7 @@ public List<Schedule> findAll() {
         return scheduleMapper.findAll();
     }
 
-
+    @Override
+    public void createTable(CreateForm form) {scheduleMapper.createTable(form);
+    }
```

Comment:
フォーマットしましょう！
セーブ時にオートフォーマットされるように設定を入れておきましょう。
```suggestion
    public void createTable(CreateForm form) {
　　    scheduleMapper.createTable(form);
    }
```

Reply:
save action設定しました。